numpy/core/setup.py

```diff
@@ -209,6 +209,32 @@ def check_prec(prec):
             else:
                 priv.extend([(fname2def(f), 1) for f in flist])
 
+            flist = [f + prec for f in C99_COMPLEX_FUNCS_CHECKED]
```

Comment:
There was some discussion of using ctypes to call these functions from python.  I didn't go down that route since we then need to find libc ourselves, but doing it this way allowed the tool chain to find it for us.  I suppose we could make a tiny module that simply exposes them to python and then use that for the tests.